spec/v2/providers/identity.spec.ts

```diff
@@ -0,0 +1,387 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2022 Firebase
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+import * as identity from '../../../src/v2/providers/identity';
+import * as identityCommon from '../../../src/common/providers/identity';
+import * as sinon from 'sinon';
+import { expect } from 'chai';
+
+const BEFORE_CREATE_TRIGGER = {
+  eventType: 'providers/cloud.auth/eventTypes/user.beforeCreate',
+  options: {
+    accessToken: false,
+    idToken: false,
+    refreshToken: false,
+  },
+};
+
+const BEFORE_SIGN_IN_TRIGGER = {
+  eventType: 'providers/cloud.auth/eventTypes/user.beforeSignIn',
+  options: {
+    accessToken: false,
+    idToken: false,
+    refreshToken: false,
+  },
+};
+
+const opts: identity.BlockingOptions = {
+  accessToken: true,
+  refreshToken: false,
+  minInstances: 1,
+  region: 'us-west1',
+};
+
+describe('identity', () => {
+  let wrapHandlerStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    wrapHandlerStub = sinon
```

Comment:
similarly - is this stubbing really needed 🤔 ? We didn't have to do something like this when testing for callable function wrapping which seems very similar to auth blocking function wrapping.

Reply:
Ah you're right, when I had the custom verification I didn't want hack it so I put a stub. Removed it in the latest commit